spec/Dockerfile.puppetdb

```diff
@@ -8,11 +8,12 @@ COPY fixtures/ssl/cert.pem /etc/puppetlabs/puppet/ssl/certs/pdb.pem
 COPY fixtures/ssl/key.pem /etc/puppetlabs/puppet/ssl/private_keys/pdb.pem
 COPY fixtures/ssl/crl.pem /etc/puppetlabs/puppet/ssl/ca/ca_crl.pem
 
-COPY fixtures/puppetdb/ssl-setup.sh /opt/puppetlabs/bin/ssl-setup.sh
+# Add out ssl-setup script in container
+COPY fixtures/puppetdb/ssl-setup.sh /docker-entrypoint.d/01-ssl-setup.sh
+run chmod +x /docker-entrypoint.d/01-ssl-setup.sh
+COPY fixtures/puppetdb/entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
 
-RUN /opt/puppetlabs/bin/ssl-setup.sh && chown -R puppetdb:puppetdb /etc/puppetlabs/puppetdb/ssl/
-
-# The normal bootstrap should copy the cert in place
-# Skip the normal bootstrapping and just run puppet-server
- ENTRYPOINT ["/opt/puppetlabs/bin/puppetdb"]
- CMD ["foreground"]
+# Use our own entrypoint
+ENTRYPOINT ["/sbin/tini", "-g", "--", "/entrypoint.sh"]
+CMD ["services"]
```

Comment:
I think you could just overwrite the `/docker-entrypoint.sh` file itself, and not have to redefine these `ENTRYPOINT` and `CWD` lines.